### ISSUE TITLE
Field 'metadata_age_start' expected a number but got ['10'].

### ISSUE THREAD

**Issue by kurama784** [unfoldadmin/django-unfold#276]
I try to making filter by one of my model IntegerField and have error
What i do wrong?

<img width="462" alt="Знімок екрана 2024-02-02 о 21 15 31" src="https://github.com/unfoldadmin/django-unfold/assets/5053208/04c9dde2-8252-4fab-b7f3-f40d0f0d6cba">

admin.py

```python
from django.contrib import admin
from .models import Payment, Client
from django.utils.translation import gettext_lazy as _
from unfold.admin import ModelAdmin
from unfold.decorators import display

from unfold.contrib.filters.admin import (
    SingleNumericFilter,
)


@admin.register(Client)
class ClientAdmin(ModelAdmin):
    list_filter_submit = True
    list_filter = (
        ("metadata_age_start", SingleNumericFilter),
    )
```

**Comment by kurama784** [comment]
Field 'metadata_age_start' expected a number but got [''].

Also this error when submit empty filter field

**Comment by lukasvinclav** [comment]
Fixed in #293
